src/extension/background-script/connectors/lndhub.ts

```diff
@@ -123,6 +125,51 @@ export default class LndHub implements Connector {
     };
   }
 
+  async getTransactions(): Promise<GetTransactionsResponse> {
+    // this connector endpoint has a fixed result set of 100 entries
+    const data = await this.request<
+      {
+        custom_records: Record<number, number>;
+        fee: number;
+        keysend: boolean;
+        memo: string;
+        payment_hash: {
+          data: number[];
+          type: string;
+        };
+        payment_preimage: string;
+        r_hash: {
+          data: number[];
+          type: string;
+        };
+        timestamp: number;
+        type: string;
+        value: number;
+      }[]
+    >("GET", "/gettxs", undefined);
```

Comment:
I think we should use the v2 endpoint (invoices/outgoing): https://ln.getalby.com/swagger/index.html#/Invoice/get_v2_invoices_outgoing

Reply:
EDIT: sorry, this would only work for Alby